script/test.sol

```diff
@@ -183,15 +185,15 @@ contract KintoDeployETHPriceIsRight is AASetup, KYCSignature, UserOp {
     IKintoWallet _newWallet;
 
     function setUp() public {
-        uint256 deployerPrivateKey = vm.envUint('PRIVATE_KEY');
+        uint256 deployerPrivateKey = vm.envUint('TEST_PRIVATE_KEY');
         vm.startBroadcast(deployerPrivateKey);
         (_kintoID, _entryPoint, _walletFactory, _sponsorPaymaster) = _checkAccountAbstraction();
         vm.stopBroadcast();
     }
 
     function run() public {
-        uint256 deployerPrivateKey = vm.envUint('PRIVATE_KEY');
-        address deployerPublicKey = vm.envAddress('PUBLIC_KEY');
+        uint256 deployerPrivateKey = vm.envUint('TEST_PRIVATE_KEY');
+        address deployerPublicKey = vm.envAddress('TEST_PUBLIC_KEY');
```

Comment:
can you check if vm.rememberKey works. ideally with that we only need one environment variable, the private key instead of two